test/functional/test_framework/test_node.py

```diff
@@ -554,7 +555,7 @@ def send_cli(self, command=None, *args, **kwargs):
         pos_args = [arg_to_cli(arg) for arg in args]
         named_args = [str(key) + "=" + arg_to_cli(value) for (key, value) in kwargs.items()]
         assert not (pos_args and named_args), "Cannot use positional arguments and named arguments in the same bitcoin-cli call"
-        p_args = [self.binary, "-datadir=" + self.datadir] + self.options
+        p_args = [self.binary, "-datadir=" + self.datadir, "-chain=" + self.chain] + self.options
```

Comment:
Why is this required, given that you already pass the datadir, which has the conf file, which sets the chain?

Reply:
yeah, it seems that's redundant, I'll remove it.